src/main/java/com/grasshouse/dorandoran/comment/service/CommentService.java

```diff
@@ -0,0 +1,60 @@
+package com.grasshouse.dorandoran.comment.service;
+
+import com.grasshouse.dorandoran.comment.domain.Comment;
+import com.grasshouse.dorandoran.comment.repository.CommentRepository;
+import com.grasshouse.dorandoran.comment.service.dto.CommentCreateRequest;
+import com.grasshouse.dorandoran.common.exception.CommentNotFoundException;
+import com.grasshouse.dorandoran.common.exception.MemberNotFoundException;
+import com.grasshouse.dorandoran.common.exception.PostNotFoundException;
+import com.grasshouse.dorandoran.member.domain.Member;
+import com.grasshouse.dorandoran.member.repository.MemberRepository;
+import com.grasshouse.dorandoran.post.domain.Post;
+import com.grasshouse.dorandoran.post.repository.PostRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class CommentService {
+
+    private final CommentRepository commentRepository;
+    private final MemberRepository memberRepository;
+    private final PostRepository postRepository;
+
+    @Transactional
+    public Long createComment(Long postId, CommentCreateRequest commentRequest) {
+        Comment comment = convertToComment(postId, commentRequest);
+        commentRepository.save(comment);
+        return comment.getId();
+    }
+
+    private Comment convertToComment(Long postId, CommentCreateRequest commentRequest) {
+        Member member = memberRepository.findById(commentRequest.getMemberId())
+            .orElseThrow(MemberNotFoundException::new);
+        Post post = postRepository.findById(postId)
+            .orElseThrow(PostNotFoundException::new);
+        Double distance = post.getLocation()
+            .calculateDistance(commentRequest.getLocation());
+
+        return Comment.builder()
+            .author(member)
+            .post(post)
+            .content(commentRequest.getContent())
+            .distance(distance)
+            .build();
+    }
+
+    @Transactional
+    public void deleteComment(Long postId, Long commentId) {
+        Comment comment = findCommentById(commentId);
+        Post post = postRepository.findById(postId)
+            .orElseThrow(PostNotFoundException::new);
```

Comment:
```suggestion
        Post post = comment.getPost();
```
메소드 인자와 url 길이를 줄일 수 있다고 생각합니다.

Reply:
저도 오구의 생각에 동의합니다~

Reply:
동의합니다! 만약 `comment.getPost()`의 값이 무조건 Post 테이블에 존재하는 값으로 보장된다면요!

다만 아직 Comment 엔티티 안에 있는 Post 필드가 nullable하면 안 된다는 조건이 없기에 저렇게 둔 것인데요,
추후에 #22 를 작업하면서 같이 수정하겠습니다.
TODO로 남겨둘게요 감사합니다 🙂